documentation/src/components/side-nav.css

```diff
@@ -21,6 +21,12 @@ aside {
     overflow: auto;
 }
 
+.scrim {
+    display: none;
+    pointer-events: none;
+    opacity: 0;
+}
```

Comment:
How do these css changes relate to the linting change?

Reply:
Indeed. I should be able to take even more of this away as we're moving from a `<div>` element here that would require special rule skipping or keyboard event handling in exchange for an `<sp-underlay>` element that already has agreement with the lint rules. Will fix!